typespec/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Wave API Documentation</title>
+    <link rel="stylesheet" href="https://unpkg.com/swagger-ui-dist/swagger-ui.css">
+</head>
+<body>
+<div id="swagger-ui"></div>
+<script src="https://unpkg.com/swagger-ui-dist/swagger-ui-bundle.js"></script>
+<script src="https://unpkg.com/swagger-ui-dist/swagger-ui-standalone-preset.js"></script>
+<script>
+    window.onload = function() {
+        const ui = SwaggerUIBundle({
+            url: "tsp-output/@typespec/openapi3/openapi.yaml",
```

Comment:
Can we change this URL so it is looking for resources under a single path of `/openapi`

```suggestion
            url: "openapi/openapi.yaml",
```

Reply:
changing it to a custom path, makes it harder to test locally the API changes, so keeping it same as typespec output path servers in every env

Reply:
This would require an additional ingress path to be created for the route `tsp-output` on the kubernetes deployment.

Reply:
Looking at Platform API Spec.

The UI is at 
https://cloud.seqera.io/openapi

With the schema being hosted at 
https://cloud.seqera.io/openapi/seqera-api-latest.yml

I still feel like we should align to 

https://wave.seqera.io/openapi/

And 

https://cloud.seqera.io/openapi/somefile.yaml

instead of hosting it on `tsp-output` especially since this change only impacts the docker container and not local dev flows.

Reply:
in the case of the platform, it's different because Openapi resides inside the platform, and in the case of Wave, it's a different app. I tried these changes but still facing the routing issue